ship_dataset.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 20 15:05:17 2019
+
+@author: j-bd
+"""
+import numpy as np
+
+from mrcnn import utils
```

Comment:
In your point of view, `mrcnn` is an external library, so it should be packed with the numpy import (same goes in other files)